slow_odgi/mygfa.py

```diff
@@ -73,15 +73,14 @@ def parse(cls, s, o) -> "Handle":
     def rev(self) -> "Handle":
         return Handle(self.name, not self.orientation)
 
-    # We need two str methods because Links and Paths prefer Handles to be
-    # string-ified in different formats.
+    """We need two str methods because Links and Paths
+    have different preferences when converting Handles to string
+    """
```

Comment:
Since this isn't "attached" to any specific variable or function, it should probably indeed just be a comment instead of a string literal (docstring).